x/gamm/types/msgs.go

```diff
@@ -18,6 +22,46 @@ const (
 	TypeMsgExitSwapShareAmountIn   = "exit_swap_share_amount_in"
 )
 
+func ValidateFutureOwner(owner string) error {
```

Comment:
```suggestion
func ValidateFutureGovernor(governor string) error {
```

Let's keep the nomenclature of governor instead of owner

Reply:
It's fixed already by you. :)